homeassistant/components/cover/mqtt.py

```diff
@@ -315,6 +339,21 @@ def async_set_cover_tilt_position(self, **kwargs):
         mqtt.async_publish(self.hass, self._tilt_command_topic,
                            level, self._qos, self._retain)
 
+    @asyncio.coroutine
+    def async_set_cover_position(self, **kwargs):
+        """Move the cover to a specific position."""
+        if ATTR_POSITION in kwargs:
+            position = kwargs[ATTR_POSITION]
+            if self._set_position_template is not None:
+                try:
+                    position = self._set_position_template.async_render()
```

Comment:
Change this to `async_render(**kwargs)`. [This will make the requested `position` from the service available to the template.](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/template.py#L90). Should also be noted in the docs.